rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -31,16 +34,14 @@ function exit_handler()
   exec 13>&-
   hdfs dfs -put /tmp/pai_yarncontainer_$CONTAINER_ID.log \
     {{{ hdfsUri }}}/Container/{{{ jobData.userName }}}/{{{ jobData.jobName }}}/log/
+  kill -9 $(docker inspect --format="{{.State.Pid}}" $docker_name)
```

Comment:
>docker inspect [](start = 12, length = 14)

actually, when a docker container runs into OOM and set oom-kill-disable, docker inspect will hang indefinitely

Reply:
removed